source/includes/changelogs/dbtools-changelogs.rst

```diff
@@ -10,38 +10,42 @@ releases.
     - Release Dates
     - MongoDB Server Support
 
-  * - :ref:`Database Tools 1.8.0 <dbtools-1.8-changelog>`
+  * - :ref:`Database Tools 100.9.0 - 100.9.5 <dbtools-100.9.0-changelog>`
```

Comment:
Does this end up linking to the 100.9.5 notes? I don't really understand the details of how this is structured.

Reply:
Yeah, it is a little confusing for me as well, but if you click the link it loads the `100.9.0` - `100.9.5` content.
A better way to test is:

1. Click the `100.7.0` release notes in the side nav
2. Click the `Database Tools 100.9.0 - 100.9.5` link at the top of the page, that should bring back the `100.9.0` RN's. 

@autarch